libs/agentc_core/agentc_core/analytics/ddls/exchanges.sqlpp

```diff
@@ -1,88 +1,85 @@
 -- Note: all_sessions.sqlpp should be run before this script.
 -- TODO (GLENN): The purpose of this view is to aid in using Ragas, but we should use the raw_logs instead.
-CREATE OR REPLACE [ANALYTICS?] FUNCTION
```

Comment:
replace the implementation of this view with the one in the notebook here: https://github.com/couchbaselabs/agent-catalog/blob/master/templates/agents/with_controlflow/notebook.ipynb

Reply:
added this under LastMessage view name - lmk if it needs to be named something else